cli/inbox/view.tsx

```
@@ -139,14 +139,18 @@ const Log: FC<LogProps> = (
   return (
     <li class={"list-group-item " + listClass}>
       <div class="d-flex w-100 justify-content-between">
-        <p class="mb-1" style="white-space: pre;">
+        <p class="mb-1" style="white-space: pre-wrap; word-break: break-word;">
           {message.map((m, i) =>
             i % 2 == 0
               ? m
               : <code>{typeof m === "string" ? m : Deno.inspect(m)}</code>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)